Windows/CommonDialog/MainWindow.xaml.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.IO;
+using Microsoft.Win32;
+using System.Diagnostics;
+using System.ComponentModel;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace CommonDialogs
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+            DataContext = new DataModel();
+        }
+
+        private void Button_OpenFileDialog(object sender, RoutedEventArgs e)
+        {
+            // Instantiate the open file dialog box.
+            // Setting the following properties are optional
+            OpenFileDialog openFileDialog = new OpenFileDialog()
+            {
+                Title = "Select any text file...",
+                InitialDirectory = @"C:\Windows",
```

Comment:
This directory does not need to exist. Also it doesn't have many text files... either use something like `Environment.GetFolderPath(SpecialFolder.MyDocuments)` or include a text file with the sample app that would be available for testing.

Reply:
Oh I see you did it for the `SaveFileDialog`... there is `SpecialFolder.Windows` then, or you can omit setting `InitialDirectory` for this one.